compilers/concrete-optimizer/concrete-optimizer/src/optimization/dag/multi_parameters/partitionning.rs

```diff
@@ -333,6 +368,269 @@ pub mod tests {
         }
     }
 
+    fn get_external_partition_index_or(
+        external_partition: &ExternalPartition,
+        p_cut: &PartitionCut,
+        default: usize,
+    ) -> PartitionIndex {
+        for (i, e_partition) in p_cut.external_partitions.iter().enumerate() {
+            if external_partition == e_partition {
+                return PartitionIndex(i + p_cut.n_internal_partitions());
+            }
+        }
+        PartitionIndex(default)
+    }
+
+    pub const GLWE_PARAMS: GlweParameters = GlweParameters {
+        log2_polynomial_size: 11,
+        glwe_dimension: 1,
+    };
+
+    pub const TFHERS_PBS_LEVEL: u64 = 1;
+
+    pub const TFHERS_MACRO_PARAMS: MacroParameters = MacroParameters {
+        glwe_params: GLWE_PARAMS,
+        internal_dim: 841,
+    };
+
+    pub fn get_tfhers_max_noise() -> f64 {
```

Comment:
it's not the max noise. It's the br noise.
Max noise is somefactor * br noise.
so there is a missing factor here (e.g. 4).